### ISSUE TITLE
WCAG2Mobile: Draft feedback - 2.1.1 Keyboard

### ISSUE THREAD

**Issue by ChrisLoiselle** [w3c/wcag2ict#658]
(Mary Jo) No comment for now...We will have to follow the MATF continuing discussions on Issue 12.

- https://w3c.github.io/matf/#success-criterion-2-1-1-keyboard
- [Read issue #12 on GitHub](https://github.com/w3c/matf/issues/12)

**Comment by pday1** [comment]
Closed - insufficient time or resource to review each WCAG2Mobile clause
